From e2e366c0b3d0d2e50bfe6908d61785f6ee321a5c Mon Sep 17 00:00:00 2001
From: "Bernhard M. Wiedemann" <bwiedemann@suse.de>
Date: Fri, 18 Mar 2022 19:51:22 +0100
Subject: [PATCH] Sort input file list

so that `epub2txt` builds in a reproducible way
in spite of indeterministic filesystem readdir order

See https://reproducible-builds.org/ for why this is good.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2013cf1..449be7f 100644
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ MANDIR  := /share/man
 APPNAME := epub2txt
 
 TARGET	:= epub2txt 
-SOURCES := $(shell find src/ -type f -name *.c)
+SOURCES := $(sort $(shell find src/ -type f -name *.c))
 OBJECTS := $(patsubst src/%,build/%,$(SOURCES:.c=.o))
 DEPS	:= $(OBJECTS:.o=.deps)
 
